### DISCUSSION TITLE
Trigger bei Öffnen eines HmIP-DLD

### DISCUSSION THREAD

**Discussion by jensrossbach** [SukramJ/aiohomematic#1331] [Q&A]
Hallo,

gibt es eine Möglichkeit, eine Automation zu triggern, wenn ein HmIP-DLD die Tür öffnet (ich meine damit nicht das Entsperren des Schlosses, sondern das Ziehen der Schlossfalle)?

**Comment by SukramJ** [comment]
Ich hab so ein Teil nicht, aber wenn ich mir die Gerätedefinition sehe gibt es nur die Zustände `locked` und `unlocked` aber nicht `open`.
Wenn man das Schloss also öffnet (Ziehen der Schlossfalle), bleibt der Zustand `unlocked`.

Alternativ schau dir doch mal an welche Events von der CCU kommen:
- Entwicklerwerkzeuge/Ereignisse öffnen und auf `state_changed` Ereignisse lauschen.
- Aktionen mit dem DLD durchführen
- Die Zum DLD gehörenden Ereignisse aus der langen Liste, die auch andere `state_change` Events herausfiltern, und schauen, was beim Öffnen passiert

**Reply by jensrossbach** [reply]
Danke für den Hinweis, schaue ich mir mal an.
